operator/controllers/reconcilers/seldon/runtime_reconciler.go

```diff
@@ -40,80 +49,124 @@ func ParseInt32(s string, defaultVal int32) (int32, error) {
 	return int32(i64), err
 }
 
-func ValidateDataflowScaleSpec(
+func getEnvVarValue(podSpec *v1.PodSpec, name string, defaultValue string) string {
+	if podSpec != nil && len(podSpec.Containers) > 0 {
+		for _, env := range podSpec.Containers[0].Env {
+			if env.Name == name {
+				return env.Value
+			}
+		}
+	}
+
+	return defaultValue
+}
+
+func min(a, b int32) int32 {
```

Comment:
please use the [builtin min function](https://go.dev/ref/spec#Min_and_max), available since go 1.21, instead